src/pages/wallet-overview/wallet-overview.html

```diff
@@ -1,46 +1,82 @@
+                <ion-card>
+                    <ion-card-header *ngFor="let currency of currentWallet.currencies;">
+                        <p class="amount-header">{{ currency.currency }}</p><p class="value-text">{{ currency.amount }}</p>
+                    </ion-card-header>
+                    <ion-card-header>
+                    <p id="total-value-text">{{ 'wallet_overview.total_value' | translate }}</p><p id="total-wallet-value">{{ currentWallet.totalCurrentCurrencyValue }}</p><p id="total-wallet-value-currency">{{ pickedCurrency }}</p>
+                    </ion-card-header>
+                    <button class="transfer-button" ion-button>Transfer</button>
+                </ion-card>
+                <ion-list>
+                    <ion-item>
+                        <ion-label>{{ 'wallet_overview.choose_value_currency' | translate }}</ion-label>
+                        <ion-select [(ngModel)]="pickedCurrency" (ionChange)="setCalculatedCurrencyValue()" interface="action-sheet">
+                        <ion-option  *ngFor="let currency of availableCurrencies" [value]="currency">{{ currency }}</ion-option>
+                        </ion-select>
+                    </ion-item>
+                </ion-list>
+                <ion-card>
+                    <ion-card-header>
+                        {{ 'wallet_overview.distribution' | translate }}
+                        <canvas #doughnutCanvas class="distribution-dougnut"> </canvas>
+                        <div id="doughnut-agenda" class="legend__list" ref="legend">
+                            <ul class="legend-list" *ngFor="let legendItem of legendList">
+                                <li>
+                                    <div class="legend-color-block" [style.backgroundColor]="legendItem.backgroundColor"></div><p class="legend-text">{{ legendItem.label }} {{ legendItem.data }}%</p>
+                                </li>
+                            </ul>
+                        </div>
+                    </ion-card-header>
+                </ion-card>
+            </ion-col>
+            <ion-col col-12 col-xl-3>
+                <h4 class="margin-header">Settings</h4>
+                <ion-card>
+                    <ion-list>
+                        <button ion-item>
+                            <ion-label>Show funds</ion-label>
+                            <ion-toggle [(ngModel)]="showFundsStatus" (ngModelChange)="fundsSwitch()" color="secondary"></ion-toggle>
+                        </button>
+
+                        <button ion-item>
+                            <ion-label>Two-Factor Authentication</ion-label>
+                            <ion-toggle [(ngModel)]="twoFactorStatus" (ngModelChange)="twoFactorStatusSwitch()" color="secondary"></ion-toggle>
+                        </button>  
+                    
+                        <button ion-item (click)="backupWalletClick()">
```

Comment:
Small point: the function you are calling has the name 'Click' in it. Strictly speaking the controller should have as little knowledge of the view as possible. Therefore it shouldn't matter to the controller how a function is called (via a click or maybe internally, it should all be the same to the controller).

I think it would be better to simply call this function 'backupWallet' or maybe 'openBackupWalletPage'.